authors/apps/articles/serializers.py

```diff
@@ -21,14 +21,32 @@ class ArticleSerializer(serializers.ModelSerializer):
     author = AuthorProfileSerializer(read_only=True)
     read_time = serializers.SerializerMethodField()
     share_links = serializers.SerializerMethodField()
-
+    favorites = serializers.SerializerMethodField()
     class Meta:
         model = Article
         fields = ('title', 'slug', 'description', 'created_at',
-                  'updated_at', 'favorited', 'favoritesCount',
+                  'updated_at', 'favorited', 'favorites', 'favoritesCount',
                   'body', 'image', 'author', 'read_time', 'share_links',
                   'likes_count', 'dislikes_count', 'average_rating')
 
+
+    def generate_usernames(self,profiles):
+        """
+        Resolve profile ids into usernames
+
+        Args:
+          profiles : Django queryset
```

Comment:
@armstrongsouljah Please follow the docstring convention shared

Reply:
Okay let me resolve these.

Reply:
Done. @ja-odur